Worktez/worktez
# fix(bugs): fixed Mailer Link and Ui

### Functionality:
Made Verification Links work
Fixed UI of theme changer button

### Solution:
Fixed the link in Verification Mail
Made the theme changer button small and moved it down 
Made it invisible for small screens for testing responsiveness
Made it invisible if the start service is not ready.

### Risk level:

### How to test:
Generate Verification Mail and check whether the link is properly generated or not
Check for the ui of Theme changer button and check if it is only visible after the startservice is ready.
